.claude-plugin/README.md

```
@@ -86,11 +86,12 @@ Call as `unraid(action="<domain>", subaction="<operation>", [params])`.
 | `logs` | Read log content (requires `log_path`; optional `tail_lines`) |
 | `flash_backup` | ⚠️ Trigger a flash backup (requires `confirm=True`) |
 
-#### `docker` — Containers (7 subactions)
+#### `docker` — Containers (8 subactions)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)